app/src/main/java/xyz/jhughes/laundry/activities/MachineActivity.java

```diff
@@ -108,12 +152,7 @@ public void createDialog() {
                 .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
                     @Override
                     public void onClick(DialogInterface dialog, int id) {
-                        SharedPreferences.Editor e = p.edit();
-                        e.putBoolean("filter", sw.isChecked());
-                        e.apply();
-                        appSectionsPagerAdapter.notifyFilterChanged();
-                        updateFilteringTextView();
-                        AnalyticsHelper.sendEventHit("Filters", AnalyticsHelper.CLICK, sw.isChecked() ? "Available only" : "All machines");
```

Comment:
https://github.com/Purdue-ACM-SIGAPP/PurdueLaundry-android/pull/113/files/518f7f1666486856566856bb4202110862099c06#diff-7833dec1f5c7ff2783b3ef072265658dR138

Its being tracked in the filtering method now to make sure its tracked everywhere.